Translations/fr.po

```diff
@@ -2067,7 +2069,7 @@ msgstr "Epée Courte"
 
 #: Source/itemdat.cpp:55 Source/itemdat.cpp:124
 msgid "Buckler"
-msgstr "Bocle"
+msgstr "Écu"
```

Comment:
The Wiki page of the buckler (https://en.wikipedia.org/wiki/Buckler) redirect to the "bocle" when looking for the french page.

The "Bocle" or "Buckler" is supposed to be small and round. An ècu could be lager with some painting.
I cannot check how is it in the game right now.

Reply:
:) I'm far from it too !